instrumentation/kafka-clients/src/main/java/brave/kafka/clients/TracingConsumer.java

```diff
@@ -313,6 +313,11 @@ public ConsumerGroupMetadata groupMetadata() {
     return delegate.groupMetadata();
   }
 
+  // Do not use @Override annotation to avoid compatibility issue version < 2.6
+  public void enforceRebalance() {
+    delegate.enforceRebalance();
```

Comment:
typical maintenance and without this we can't really trace kafka 2.6 as it won't completely implement the interface